PCbuild/get_external.py

```diff
@@ -53,7 +55,20 @@ def main():
         verbose=args.verbose,
     )
     final_name = args.externals_dir / args.tag
-    extract_zip(args.externals_dir, zip_path).replace(final_name)
+    extracted = extract_zip(args.externals_dir, zip_path)
+    for _ in range(5):
+        try:
+            extracted.replace(final_name)
+            break
+        except PermissionError as ex:
+            print(f"Encountered permission error '{ex}'. Retrying in five seconds...", file=sys.stderr)
+            time.sleep(5)
```

Comment:
If this is the final try, the message is a lie and it sleeps for 5 seconds before exiting.